specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2023-05-01-preview/extensionTypes.json

```diff
@@ -0,0 +1,766 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{clusterRp}/{clusterResourceName}/{clusterName}/providers/Microsoft.KubernetesConfiguration/extensionTypes": {
+      "get": {
+        "tags": [
+          "ExtensionTypes",
+          "ClusterExtensionTypes"
+        ],
+        "description": "List installable Extension Types for the cluster based region and type for the cluster.",
+        "operationId": "ExtensionTypes_List",
+        "x-ms-examples": {
+          "Get Extension Types": {
+            "$ref": "./examples/ListExtensionTypes.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/2023-05-01-preview/parameters.json#/parameters/ClusterRpParameter"
+          },
+          {
+            "$ref": "../../common/2023-05-01-preview/parameters.json#/parameters/ClusterResourceNameParameter"
+          },
+          {
+            "$ref": "../../common/2023-05-01-preview/parameters.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "name": "publisherId",
+            "in": "query",
+            "description": "Filter results by Publisher ID of a marketplace extension type",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "offerId",
+            "in": "query",
+            "description": "Filter results by Offer or Product ID of a marketplace extension type",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "planId",
+            "in": "query",
+            "description": "Filter results by Plan ID of a marketplace extension type",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "releaseTrain",
+            "in": "query",
+            "description": "Filter results by release train (default value is stable)",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ExtensionTypesList"
```

Comment:
You are returning a list. We expect tis should support pagination

Reply:
I must have missed that one. It should be there now